src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -2069,46 +2063,24 @@ public void run() {
 		};
 	}*/
 	
+	public boolean noJump(String bName) {
+		for(String str:ignoreJump) {
+			if(bName.contains(str)) {return true;}
```

Comment:
Is this quite foolproof? Is there some edge case where maybe you don't want to jump for a "STEP" or something but do for a "DOUBLE_STEP"? (Bad example since it doesn't apply, but a similar idea where on material name is included in another)

Reply:
It may be okay, but changing the 'it matches' check to an 'it contains' checks seems problematic

Reply:
Maybe check GRASS and GRASS_BLOCK?

Reply:
See main comment